.github/workflows/singledb-version-upgrade.yml

```diff
@@ -0,0 +1,139 @@
+name: Major Version Upgrade Tests for singledb mode
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-mvu-tests-singledb:
+    env:
+      env:
+      OLD_INSTALL_DIR: postgres13
+      NEW_INSTALL_DIR: postgres14
+      ENGINE_BRANCH_FROM: BABEL_1_X_DEV__PG_13_6
+      EXTENSION_BRANCH_FROM: BABEL_1_X_DEV
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install Dependencies
```

Comment:
can we put the below logic in a composite action and share between both single and multi db

Reply:
We need the ``checkout@v2`` action because the workflow yaml file uses the path `./github/composite-actions/ca` to reference the action , but the install dependencies step can be moved to the setup-base-version ca , changed in the lates commit